<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1100</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>680</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;This Project is Created by:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Mr : Jose Rey&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;&lt;br/&gt;Please do support him through a star in github repository&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/icon_info.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/icon_logo.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMode">
   <property name="geometry">
    <rect>
     <x>1054</x>
     <y>0</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
background:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../images/icon_night_mode.png</normaloff>../images/icon_night_mode.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>1101</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@900&amp;display=swap');

#lblTitle{
background:transparent;
color: #626262;
font-family: 'Poppins', sans-serif;
font-weight:900;
font-size: 36px;
font-style: normal;
line-height: normal;
}</string>
   </property>
   <property name="text">
    <string>Facebook Video Downloader</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFacebook">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>180</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
background:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../images/icon_facebook.png</normaloff>../images/icon_facebook.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGithub">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>180</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
background:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../images/icon_github.png</normaloff>../images/icon_github.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnYoutube">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>180</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
background:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../images/icon_youtube.png</normaloff>../images/icon_youtube.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputLink">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>720</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400&amp;display=swap');

#inputLink{
border-radius: 8px;
border: 1px solid #D4D4D4;
background: #EFEFEF;
color: #000000;
font-family: 'Poppins', sans-serif;
font-size: 12px;
font-style: normal;
font-weight: 400;
line-height: normal;
padding-left:5px
}</string>
   </property>
   <property name="placeholderText">
    <string>Enter Facebook Video URL</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblNote">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>721</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400&amp;display=swap');

#lblNote{
background:transparent;
color: #6F6F6F;
font-family: 'Poppins', sans-serif;
font-size: 12px;
font-style: normal;
font-weight: 400;
line-height: normal;
}</string>
   </property>
   <property name="text">
    <string>Note: Certain videos may be undownloadable due to privacy settings. For details, visit the GitHub page.</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorMessage">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>315</y>
     <width>721</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400&amp;display=swap');

#errorMessage{
color: rgb(222, 34, 43);
font-family: 'Poppins', sans-serif;
font-size: 12px;
font-style: normal;
font-weight: 400;
line-height: normal;
}</string>
   </property>
   <property name="text">
    <string>Error Message</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSearch">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>360</y>
     <width>245</width>
     <height>38</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400&amp;display=swap');

#btnSearch{
border-radius: 10px;
background: #108AE3;
color: #FFF;
font-family: 'Poppins', sans-serif;
font-size: 16px;
font-style: normal;
font-weight: 400;
line-height: normal;
}
#btnSearch:hover{
background:#0093FF;
}</string>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDownload">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>620</y>
     <width>245</width>
     <height>38</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400&amp;display=swap');

#btnDownload{
border-radius: 10px;
background: #108AE3;
color: #FFF;
font-family: 'Poppins', sans-serif;
font-size: 16px;
font-style: normal;
font-weight: 400;
line-height: normal;
}
#btnDownload:hover{
background:#0093FF;
}</string>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblNote_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>590</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400&amp;display=swap');

#lblNote_2{
color: #111111;
font-family: 'Poppins', sans-serif;
font-size: 12px;
font-style: normal;
font-weight: 400;
line-height: normal;
}</string>
   </property>
   <property name="text">
    <string>Congratulations!!! Video Found</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="container" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-1</y>
     <width>1161</width>
     <height>721</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#container{
background: #FFFFFF;
}</string>
   </property>
   <widget class="QLabel" name="imgThumbnail">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>430</y>
      <width>300</width>
      <height>150</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>150</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>150</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/icon_thumbnail_day.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <zorder>container</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>btnMode</zorder>
  <zorder>lblTitle</zorder>
  <zorder>btnFacebook</zorder>
  <zorder>btnGithub</zorder>
  <zorder>btnYoutube</zorder>
  <zorder>inputLink</zorder>
  <zorder>lblNote</zorder>
  <zorder>errorMessage</zorder>
  <zorder>btnSearch</zorder>
  <zorder>btnDownload</zorder>
  <zorder>lblNote_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
